{
    "config": {
        "abort": {
            "already_configured": "Za\u0159\u00edzen\u00ed je ji\u017e nastaveno",
            "reauth_successful": "Op\u011btovn\u00e9 ov\u011b\u0159en\u00ed bylo \u00fasp\u011b\u0161n\u00e9",
            "reconfigure_successful": "P\u0159enastaven\u00ed bylo \u00fasp\u011b\u0161n\u00e9"
        },
        "error": {
            "cannot_connect": "Nepoda\u0159ilo se p\u0159ipojit",
            "invalid_auth": "Neplatn\u00e9 ov\u011b\u0159en\u00ed",
            "unknown": "Neo\u010dek\u00e1van\u00e1 chyba"
        },
        "step": {
            "reauth_confirm": {
                "data": {
                    "password": "Heslo",
                    "username": "U\u017eivatelsk\u00e9 jm\u00e9no"
                },
                "title": "Platnost ov\u011b\u0159en\u00ed pro {name} vypr\u0161ela"
            },
            "reconfigure_confirm": {
                "data": {
                    "host": "Hostitel",
                    "password": "Heslo",
                    "port": "Port",
                    "ssl": "Pou\u017e\u00edv\u00e1 SSL certifik\u00e1t",
                    "username": "U\u017eivatelsk\u00e9 jm\u00e9no",
                    "verify_ssl": "Ov\u011b\u0159it certifik\u00e1t SSL"
                },
                "data_description": {
                    "host": "N\u00e1zev hostitele nebo IP adresa za\u0159\u00edzen\u00ed, na kter\u00e9m je spu\u0161t\u011bna va\u0161e instance pyLoad.",
                    "port": "pyLoad standardn\u011b pou\u017e\u00edv\u00e1 port 8000."
                }
            },
            "user": {
                "data": {
                    "host": "Hostitel",
                    "name": "N\u00e1zev",
                    "password": "Heslo",
                    "port": "Port",
                    "ssl": "Pou\u017e\u00edv\u00e1 SSL certifik\u00e1t",
                    "username": "U\u017eivatelsk\u00e9 jm\u00e9no",
                    "verify_ssl": "Ov\u011b\u0159it certifik\u00e1t SSL"
                },
                "data_description": {
                    "host": "N\u00e1zev hostitele nebo IP adresa za\u0159\u00edzen\u00ed, na kter\u00e9m je spu\u0161t\u011bna va\u0161e instance pyLoad.",
                    "name": "N\u00e1zev, kter\u00fd se m\u00e1 pou\u017e\u00edt pro va\u0161i instanci pyLoad v Home Assistantovi.",
                    "port": "pyLoad standardn\u011b pou\u017e\u00edv\u00e1 port 8000."
                }
            }
        }
    },
    "entity": {
        "button": {
            "abort_downloads": {
                "name": "P\u0159eru\u0161it v\u0161echna b\u011b\u017e\u00edc\u00ed stahov\u00e1n\u00ed"
            },
            "delete_finished": {
                "name": "Smazat dokon\u010den\u00e9 soubory/bal\u00ed\u010dky"
            },
            "restart": {
                "name": "Restartovat j\u00e1dro pyload"
            },
            "restart_failed": {
                "name": "Restartovat v\u0161echny ne\u00fasp\u011b\u0161n\u00e9 soubory"
            }
        },
        "sensor": {
            "active": {
                "name": "Aktivn\u00ed stahov\u00e1n\u00ed"
            },
            "free_space": {
                "name": "Voln\u00e9 m\u00edsto"
            },
            "queue": {
                "name": "Stahov\u00e1n\u00ed ve front\u011b"
            },
            "speed": {
                "name": "Rychlost"
            },
            "total": {
                "name": "Celkov\u00e1 stahov\u00e1n\u00ed"
            }
        },
        "switch": {
            "download": {
                "name": "Pozastavit/Obnovit frontu"
            },
            "reconnect": {
                "name": "Automatick\u00e9 op\u011btovn\u00e9 p\u0159ipojen\u00ed"
            }
        }
    },
    "exceptions": {
        "service_call_auth_exception": {
            "message": "Nelze odeslat p\u0159\u00edkaz do pyLoad kv\u016fli chyb\u011b ov\u011b\u0159ov\u00e1n\u00ed, zkuste to znovu pozd\u011bji"
        },
        "service_call_exception": {
            "message": "Nelze odeslat p\u0159\u00edkaz do pyLoad kv\u016fli chyb\u011b p\u0159ipojen\u00ed, zkuste to znovu pozd\u011bji"
        },
        "setup_authentication_exception": {
            "message": "Ov\u011b\u0159en\u00ed u\u017eivatele {username} se nezda\u0159ilo, ov\u011b\u0159te sv\u00e9 p\u0159ihla\u0161ovac\u00ed \u00fadaje"
        },
        "setup_parse_exception": {
            "message": "Nelze analyzovat data z pyLoad API, zkuste to znovu pozd\u011bji"
        },
        "setup_request_exception": {
            "message": "Nelze se p\u0159ipojit a na\u010d\u00edst data z pyLoad API, zkuste to znovu pozd\u011bji"
        }
    },
    "issues": {
        "deprecated_yaml_import_issue_cannot_connect": {
            "description": "Nastaven\u00ed pyLoad pomoc\u00ed YAML se odeb\u00edr\u00e1, ale p\u0159i pokusu o import nastaven\u00ed YAML do\u0161lo k chyb\u011b p\u0159ipojen\u00ed.\n\nOv\u011b\u0159te, \u017ee m\u00e1te stabiln\u00ed p\u0159ipojen\u00ed k internetu a restartujte Home Assistanta, abyste to zkusili znovu, nebo odeberte nastaven\u00ed pyLoad YAML ze souboru configuration.yaml a pokra\u010dujte v [nastaven\u00ed integrace]({url}) ru\u010dn\u011b.",
            "title": "Import konfigurace pyLoad YAML se nezda\u0159il"
        },
        "deprecated_yaml_import_issue_invalid_auth": {
            "description": "Nastaven\u00ed pyLoad pomoc\u00ed YAML se odeb\u00edr\u00e1, ale p\u0159i pokusu o import nastaven\u00ed YAML do\u0161lo k chyb\u011b ov\u011b\u0159ov\u00e1n\u00ed.\n\nOpravte nastaven\u00ed YAML a restartujte Home Assistanta, abyste to zkusili znovu, nebo odeberte nastaven\u00ed pyLoad YAML ze souboru configuration.yaml a pokra\u010dujte v [nastaven\u00ed integrace]({url}) ru\u010dn\u011b.",
            "title": "Import konfigurace pyLoad YAML se nezda\u0159il"
        },
        "deprecated_yaml_import_issue_unknown": {
            "description": "Nastaven\u00ed pyLoad pomoc\u00ed YAML se odeb\u00edr\u00e1, ale p\u0159i pokusu o import nastaven\u00ed YAML do\u0161lo k nezn\u00e1m\u00e9 chyb\u011b.\n\nUjist\u011bte se, \u017ee nastaven\u00ed YAML je spr\u00e1vn\u00e9 a restartujte Home Assistanta, abyste to zkusili znovu, nebo odeberte nastaven\u00ed YAML ze souboru configuration.yaml a pokra\u010dujte v [nastaven\u00ed integrace]({url}) ru\u010dn\u011b.",
            "title": "Import konfigurace pyLoad YAML se nezda\u0159il"
        }
    }
}